### PR TITLE
THREESCALE-11011: Authorize.net is a living dead 🧟‍♀️

### PR DESCRIPTION
**Pull request by jlledom**
**What this PR does / why we need it**:

Reverts 65a284e0, afa1b52d and a31cd401

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-11011

### CODE REVIEW

**Review by josemigallas — DISMISSED** [review]

**Review by lvillen — DISMISSED** [review]

File: lib/developer_portal/lib/liquid/drops/payment_gateway.rb [hunk 1]
```diff
@@ -6,11 +6,16 @@ def initialize(account)
         @account = account
       end
 
-      desc "Returns whether current payment gateway is Braintree."
+      desc "Returns whether current payment gateway is authorize.Net."
```
Comment (akostadinov) [inline lib/developer_portal/lib/liquid/drops/payment_gateway.rb:9]: this is wrong

File: test/unit/liquid/drops/urls_drop_test.rb [hunk 2]
```diff
@@ -12,8 +12,8 @@ def setup
     @provider.update_attribute(:payment_gateway_type, 'bogus')
     assert_nil @drop.payment_details
 
-    @provider.update_attribute(:payment_gateway_type, 'stripe')
-    assert_equal '/admin/account/stripe',  @drop.payment_details.to_str
+    @provider.update_attribute(:payment_gateway_type, 'authorize_net')
+    assert_equal '/admin/account/authorize_net',  @drop.payment_details.to_str
```
Comment (akostadinov) [inline test/unit/liquid/drops/urls_drop_test.rb:16]: perhaps keep the `stripe` assertion, no need to delete it afaict

File: lib/developer_portal/lib/liquid/drops/payment_gateway.rb (hunk 1 above)
Comment (mayorova) [reply lib/developer_portal/lib/liquid/drops/payment_gateway.rb:9]: It is, although it was like that originally :sweat_smile:  https://github.com/3scale/porta/commit/a31cd4011bba74b7bbdb152db549980f1fd5f35c#diff-0402b588da3aab79767114e1baea7824c5dacc66a00ae9f73492fe612fb03494

**Review by mayorova — DISMISSED** [review]

File: lib/developer_portal/config/initializers/liquid.rb [hunk 3]
```diff
@@ -25,7 +25,6 @@
     Liquid::Tags::AuthorizeNetForm,
     Liquid::Tags::PaymentExpressForm,
     Liquid::Tags::BraintreeCustomerForm,
-    Liquid::Tags::OgoneForm,
```
Comment (mayorova) [inline lib/developer_portal/config/initializers/liquid.rb]: @akostadinov I think this is not right

we previously had issues with removing that Liquid, so we brought it back.

Comment (jlledom) [reply lib/developer_portal/config/initializers/liquid.rb]: @mayorova that's right

**Review by mayorova — APPROVED** [review]

File: lib/developer_portal/config/initializers/liquid.rb (hunk 3 above)
Comment (akostadinov) [reply lib/developer_portal/config/initializers/liquid.rb]: didn't we live for 1 month without it?

Comment (mayorova) [reply lib/developer_portal/config/initializers/liquid.rb]: @akostadinov nope, we removed it, then realized that it breaks some existing portals, so we rolled it back. the Authorize.net removals did NOT touch the OgoneForm liquid.

Comment (jlledom) [reply lib/developer_portal/config/initializers/liquid.rb]: @akostadinov this is how it looks currently https://github.com/3scale/porta/blob/a31cd4011bba74b7bbdb152db549980f1fd5f35c/lib/developer_portal/config/initializers/liquid.rb

**Pull request merged**